Translated "Dato" to "Date" (#565)
Altinn/altinn-studio-docs: content/app/design/guidelines/components/date/_index.en.md

---
title: Date

> description: The date picker is used when selecting a time close to the present date. To select dates that are far in the past, or in the future, a datefield should be used.

Reviewer: ```suggestion
description: The date picker can be used when selecting a (relatively) recent date. To select dates that are far in the past (or future) a date field can be used instead.
```

weight: 40
toc: true
---

## Date picker

> You can use the date picker if something is being planned, or for selecting a recent or close time.

Reviewer: ```suggestion
Use the date picker if something is being planned, or for selecting a (relatively) recent or close future date.
```

### Guidelines:

> -  The date picker is not recommended if your goal is to collect birthdates or other datest further back in time. Alternatively, look at studies from [gov.uk](https://design-system.service.gov.uk/patterns/dates/#asking-for-memorable-dates) 

Reviewer: ```suggestion
-  The date picker is not recommended if your goal is to collect birth dates or other dates further back in time (see studies from [gov.uk](https://design-system.service.gov.uk/patterns/dates/#asking-for-memorable-dates)) 
```

> -  In these cases you should instead use [datofeltet](#datofelt).

Reviewer: ```suggestion
-  In these cases you should instead use a [date field](#date-field).
```

> -  Notice correct validation and give the user information about what are valid and invalid dates. Should the user be able to select dates further back in time? Within a given period?

Reviewer: ```suggestion
-  Add correct validation and give the user information about what are valid and invalid dates. Should the user be able to select dates further back in time? Within a given period?
```

### Example of usage:

<iframe style="border: 3px solid rgb(0 0 0 / 90%);border-radius: 9px;" width="100%" height="550" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FwnBveAG2ikUspFsQwM3GNE%2FPrototyping-av-skjematjenester%3Fnode-id%3D4833%253A961" allowfullscreen></iframe>

---

## Date field
Date fields should be used for dates further back in time, or for dates that the user knows well. Three textfields is the simplest way to take in dates in these cases. If someone is planning something and a specific day (monday, friday, etc.) is important the datepicker should be used instead.

{{% panel theme="warning" %}} ⚠️ This component is not accessible in Altinn Studio yet. 
{{% /panel %}}

### Guidelines:

> - Always lable the input boxes (day, month, year), and keep a leading text for the input group. 

Reviewer: ```suggestion
- Always label the input boxes (day, month, year), and add a label/heading for the input group as well. 
```
